### ISSUE TITLE
updateCustomParams iOS Version 5.0.3

### ISSUE THREAD

**Issue by wwwdata** [mapp-digital/Mapp-Intelligence-Flutter-Tracking#1]
Hello, after upgrading to Version `5.0.3` it seems the native dependency is incompatible with the dart package because types mismatch.

```
flutter: type 'String' is not a subtype of type 'FutureOr<bool>?'
flutter:
#0      PluginMappintelligence.updateCustomParams (package:plugin_mappintelligence/plugin_mappintelligence.dart:204:25)
<asynchronous suspension>
#1      PluginMappintelligence.build (package:plugin_mappintelligence/plugin_mappintelligence.dart:152:5)
```

is what we get after initializing the plugin.. It looks like the following code is returning a `String` and not the expected `bool` from the native MethodChannel `invokeMethod` call.

https://github.com/mapp-digital/Mapp-Intelligence-Flutter-Tracking/blob/923bce262f6e2f382639be08627591690a8f5e25/lib/plugin_mappintelligence.dart#L198-L205

**Comment by christofkost** [comment]
I have the same issue, which probably results in the app randomly crashing during startup on iOS.

**Comment by StefanMiticon** [comment]
Hi all, 

please try latest version, just to confirm that it is resolved.

**Comment by wwwdata** [comment]
Steps to reproduce:

- `flutter create` a new app
- `flutter pub add plugin_mappintelligence`
- then try to start the app it will give you my mentioned error message.

I just tested this today with a brand new fresh Flutter App and only had `plugin_mappintelligence` added as the sole dependency with it's latest version `5.0.4+1`

**Comment by stellatores** [comment]
We published version 5.0.4+2 to solve the CFBundle error, please try with that and let us know if that solved the issue.

**Comment by wwwdata** [comment]
Thank you it is working now and also the initial error is not happening anymore with the custom params 👍
